host/CMakeLists.txt

```diff
@@ -342,6 +342,21 @@ list(
   strings.c
   traceh_enclave.c)
 
+# Add MSVC compile flag /Z7 to enable debugging into OE SDK source from
+# applications that use release builds.
+if (OE_SGX AND WIN32)
+  foreach (file ${PLATFORM_HOST_ONLY_SRC})
```

Comment:
I'd prefer adding it to the target as Ming-Wei suggested instead of for each file.

Can you find out what compile options cmake adds by default for various  OE SDK build types (Release, RelWithDebInfo, Debug) on Windows? We need to make sure that /Z7 takes precedence over other debug flags cmake adds.